### PR TITLE
431 remove button chaos on event creation page

### PR DESCRIPTION
**Pull request by nstrelow**
refs: #431 

remove kind
hidden now is displayed as choice between "public" and "private" for the user

### CODE REVIEW

**Review by nilsthamm — APPROVED** [review]

**Comment by Baschdl** [comment]
@nstrelow (@nilsthamm) You introduced a DEPRECATION WARNING in our tests with this test
`DEPRECATION WARNING: The assertion was not run because of an invalid css selector.
unexpected '"event[hidden]"' after '[' (called from block (3 levels) in <top (required)> at /home/travis/build/hpi-swt2/workshop-portal/spec/views/events/new.html.erb_spec.rb:17)
DEPRECATION WARNING: The assertion was not run because of an invalid css selector.
unexpected '"event[published]"' after '[' (called from block (3 levels) in <top (required)> at /home/travis/build/hpi-swt2/workshop-portal/spec/views/events/new.html.erb_spec.rb:18)`

File: app/views/events/participants.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
           </th>
         <% end %>
         <% if can? :show_eatings_habits, Event %>
-          <th>
+          <th>7
```
Comment (bjrne) [inline app/views/events/participants.html.erb:26]: Wait, is this 7 wanted here?

File: config/locales/de.events.yml [hunk 2]
```diff
@@ -159,8 +158,9 @@ de:
             one: "Zeitspanne"
             other: "Zeitspannen"
           application_deadline: "Bewerbungsschluss"
-          published: "Öffentlich"
+          published: "Entwurf"
```
Comment (bjrne) [inline config/locales/de.events.yml:161]: This seems to be the exact opposite of what I'd expect `published` to mean.

File: spec/features/event_spec.rb [hunk 3]
```diff
@@ -110,13 +110,13 @@
   end
 
   describe "create page" do
-    ['Camp', 'Workshop'].each do |kind|
-      it "should allow picking the #{kind} kind camp" do
+    I18n.t(".events.type").each do |type|
+      it "should allow picking the #{type[1]} type" do
```
Comment (bjrne) [inline spec/features/event_spec.rb:114]: Is `type` really an array here?

**Review by bjrne — COMMENTED** [review]
@nstrelow (@nilsthamm) I noticed some minor things that you might want to see/adress.

File: spec/features/event_spec.rb (hunk 3 above)
Comment (nstrelow) [reply spec/features/event_spec.rb:114]: yes, the locales are a array of length 2 with the string and the symbol to it

File: config/locales/de.events.yml (hunk 2 above)
Comment (nstrelow) [reply config/locales/de.events.yml:161]: then I will rename it to draft

**Pull request merged**